workshop_app/forms.py

```diff
@@ -163,7 +184,7 @@ class ProposeWorkshopDateForm(forms.ModelForm):
     """
     Coordinators will propose a workshop and date 
     """ 
-        
+    errorlist_css_class = 'errorlist'
```

Comment:
incase of invalid or missing data, this class is called! 
errorlist is the div's class name  and I am using that to change it's default color(i.e black) to red.